db/seeds.rb

```diff
@@ -6,13 +6,29 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+if Rails.env.test?
```

Comment:
Please delete this block -- if the Rails environment is test, you're defining some places but not doing anything with them.

Reply:
Actually, now I see that you're defining different places depending on env, then creating them at the end of this file -- remember that seeding is really only for dev or _production_ (mostly dev). Let me know if there's any reason you couldn't just delete the test conditional and move the places out of the dev conditional.